docs/requirements.md

```diff
@@ -1,11 +1,14 @@
 # Requirements
 
+Please note that in order to deploy resources from this repo successfully, it is necessary to meet the below requirements as the terraform code we provide will not help create them
```

Comment:
So that this is as short and clear as possible, suggest changing to: 
In order to deploy resources from this repo successfully, it is necessary to meet the requirements below, as the terraform code we provide will not create them

Reply:
thank you, updated this